### PR TITLE
Replace Open RPC in footer and JSON-RPC doc, fix numerous links throu…

### PR DESCRIPTION
**Pull request by Clay-Mysten**
…ghout

### CODE REVIEW

File: doc/src/build/json-rpc.md [hunk 1]
```diff
@@ -40,17 +40,20 @@ export SUI_RPC_HOST=http://127.0.0.1:5001
 
 ## Sui software development kits
 
-You may sign transactions and interact with the Sui network using:
+You may sign transactions and interact with the Sui network using the:
```
Comment (randall-Mysten) [inline doc/src/build/json-rpc.md:43]:
```suggestion
You can sign transactions and interact with the Sui network using any of the following:
```

File: doc/src/build/json-rpc.md [hunk 2]
```diff
@@ -63,7 +66,7 @@ curl --location --request POST $SUI_RPC_HOST \
 --data-raw '{ "jsonrpc":"2.0", "method":"rpc.discover","id":1}'
 ```
 
-You can see an example of the discovery service in the [OpenRPC Playground](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/MystenLabs/sui/main/crates/sui-open-rpc/spec/openrpc.json).
+You can see an example of the discovery service in the [Sui API Reference](https://docs.sui.io/sui-jsonrpc).
```
Comment (randall-Mysten) [inline doc/src/build/json-rpc.md:69]: is it this topic?
https://docs.sui.io/devnet/build/json-rpc#rpcdiscover

**Review by randall-Mysten — APPROVED** [review]

File: doc/src/build/json-rpc.md (hunk 2 above)
Comment (Clay-Mysten) [reply doc/src/build/json-rpc.md:69]: > is it this topic? https://docs.sui.io/devnet/build/json-rpc#rpcdiscover

That may well be going away. Leaving for @patrickkuo and myself to discuss.

Comment (Clay-Mysten) [reply doc/src/build/json-rpc.md]: Removed.

**Pull request merged**